cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/AwsStepfunctionPublishWorker.java

```diff
@@ -380,6 +379,16 @@ private String getProgressAsJson(PublishJobProgress progress) {
 		return progressJson;
 	}
 	
+	private void sendTaskResultBestEffort(GetActivityTaskResult taskResult, CommandRuntimeException cre) {
+		try {
+			sendTaskResult(taskResult, cre);
+		} catch (TaskTimedOutException timedOutEx) {
+			logger.error("AWS Client has timed out while tring to send task result", timedOutEx);
```

Comment:
No, you have not understood TTE. 

AWS Task has timed out while processed by the Worker (heartbeats too sparse or total time exceeded).